src/services/patient.ts

```diff
@@ -739,8 +739,8 @@ export const fetchPatient = async (patientId: string, groupId?: string): Promise
         `/Condition?patient=${patientId}&_sort=-recorded-date&size=20${groupFilter}`
       ),
       api.get<FHIR_API_Response<IClaim>>(`/Claim?patient=${patientId}&_sort=-created&size=20${groupFilter}`),
-      api.get<FHIR_API_Response<IComposition>>(
-        `/Composition?patient=${patientId}&size=20&_sort=-date&status=final&_elements=status,type,encounter,date,title${groupFilter}`
+      api.get<FHIR_API_Response<IDocumentReference>>(
+        `/DocumentReference?patient=${patientId}&size=20&_sort=-date&status=final&_elements=status,type,encounter,date,title${groupFilter}`
```

Comment:
have you tested this ?

Reply:
It won't work indeed. It's part of the aphp context but since I changed the typing, i get a compilation error if I don't change the type here.